app/src/main/java/de/xikolo/controllers/dialogs/CreateTicketDialog.kt

```diff
@@ -124,18 +132,28 @@ class CreateTicketDialog : ViewModelDialogFragment<TicketViewModel>(), HelpdeskT
             getButton(AlertDialog.BUTTON_NEGATIVE).setOnClickListener {
                 cancel(this)
             }
+            getButton(AlertDialog.BUTTON_POSITIVE).setOnClickListener {
+                if (context.isOnline) {
+                    viewModel.send(titleEditText.text.toString(), messageEditText.text.toString(), topic, emailEditText.text.toString(), courseId)
+                    (dialog as AlertDialog).dismiss()
```

Comment:
Is it always safe to cast the dialog to an `AlertDialog`? Otherwise a safe cast `(dialog as? AlertDialog)?.dismiss()` can be used.

Reply:
Actually it does not need to be casted at all in this case but the question still applies for line 131.

Reply:
It is not always called as AlertDialog therefore I will implement the safe cast